README.md

```diff
@@ -76,7 +103,21 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('/'),function(req,res) {
+  res.send('hello world')
+}
 
+app.post('/'),function(res,req){
+  res.send('my name is luisa')
+}
+
+app.put('/'),function(res,req){
+  res.send('good morning')
+}
+
+app.patch('/'),function(req,res){
+  res.send('nice to meet you')
+}
```

Comment:
Got most of the request methods but did not include the RESTful action associated